src/Containers/CadView.jsx

```diff
@@ -193,6 +193,17 @@ export default function CadView({
   const setAlertMessage = (msg) =>
     setAlert(<Alert onCloseCb={() => navToDefault(navigate, appPrefix)} message={msg}/>)
 
+  /**
+   * Prevent reloading page without user approval when loading a model from local
+   *
+   * @param {Event} e
+   * @return {string} confirmationMessage
+   */
+  function handleBeforeUnload(e) {
+    const confirmationMessage = 'It looks like you have been editing something. If you leave before saving, your changes will be lost.';
```

Comment:
Please use: 'Uploaded models are not saved.  If you leave this page you will need to upload the model again.'

Reply:
I have done.